layoutlm/scripts/funsd_preprocess.py

```diff
@@ -107,9 +107,9 @@ def seg_file(file_path, tokenizer, max_len):
             if current_subwords_len == 0:
                 continue
 
-            if (subword_len_counter + current_subwords_len) > max_len:
+            if (subword_len_counter + current_subwords_len) >= max_len:
                 fw_p.write("\n" + line + "\n")
-                subword_len_counter = 0
+                subword_len_counter += current_subwords_len
```

Comment:
Yes, the first `current_subwords_len` is ignored when creating a new sequence. But I think the correct operation should be `subword_len_counter = current_subwords_len`.

Reply:
Yes, you're right, removed the addition.

Reply:
Much appreciated if you can reconsider and update L110 so that I can merge this pr.

Reply:
I've updated L110, thanks for the review.